Makefile

```
@@ -12,7 +12,26 @@ help: ## Show available targets
+preflight: fmt-check clippy test deny audit typos verify-no-network-default ## Run all CI checks locally
+
+# --- Architectural-boundary enforcement ---------------------------------------
+#
+# CONSTITUTION.md section 6.1 (post-Stage-B amendment) requires
+# that the default build contains no network-capable code. The
+# `install` Cargo feature gates `ureq`, `tar`, and `dirs`; the
+# gzip decoder and TOML parser crates were already transitive via
+# typst before Stage B. This target fails if `ureq`, `tar`, or
+# `dirs` ever leak into the default dependency graph.
+
+verify-no-network-default: ## Fail if ureq/tar/dirs leak into the default build
```

checkmake (0.3.2)

[warning] Target body for "verify-no-network-default" exceeds allowed length of 5 lines (8).

(maxbodylength)